src/sagemaker/fw_utils.py

```diff
@@ -148,7 +148,15 @@
 ]
 
 
-TORCH_DISTRIBUTED_SUPPORTED_FRAMEWORK_VERSIONS = ["1.11", "1.11.0"]
+TORCH_DISTRIBUTED_TRAINIUM_SUPPORTED_FRAMEWORK_VERSIONS = [
```

Comment:
Instead of listing all supported versions, shall we set a min supported version and compare with it?

Reply:
I think we can't do this since user may also put some version like `1.11.1` which we don't support. 
And this way of version check is aligned with other version checking in this file.